packages/react-components/components/PhoneNumberInput.tsx

```diff
@@ -65,6 +71,33 @@ export default class PhoneNumberInput extends React.Component<Props, State> {
     }
   }
 
+  async triggerPhoneNumberRequest() {
+    try {
+      const phone = await SmsRetriever.requestPhoneNumber()
+      const phoneNumber = parsePhoneNumberFromString(phone, undefined)
```

Comment:
Please use the existing utils in `@celo/utils/src/phoneNumbers` which add a bunch of error handling and handle special cases we've encountered for certain countries